Userland/Libraries/LibGfx/ICCProfile.cpp

```diff
@@ -77,7 +77,7 @@ struct ICCHeader {
     BigEndian<u16> profile_version_zero;
 
     BigEndian<DeviceClass> profile_device_class;
-    BigEndian<u32> data_color_space;
+    BigEndian<ColorSpace> data_color_space;
     BigEndian<u32> pcs; // "Profile Connection Space"
```

Comment:
This isn't related to your change, so not asking you to do anything now, but I feel like this var should just be called `profile_connection_space` instead of having a comment say what it is

Reply:
It uses the names used in the spec. It'll be called "profile connection space" in the class's public API, but I thought we try to use names from the spec where appropriate? That's what I was trying to do here. (See link above struct)

Reply:
I personally prefer actually useful names over spec names. If I saw `pcs` in the implementation, my first thought would be 'the heck does this stand for'

Reply:
A'ight, will change when I add the PCS dumping code.